@navikt/core/react/src/util/i18n/i18n.context.ts

```diff
@@ -58,3 +65,18 @@ export function useI18n<T extends Component>(
 
   return translate;
 }
+
+export function useDateLocale() {
+  const languageProviderContext = useContext(LanguageProviderContext);
+  const i18n = languageProviderContext.translations;
+  const i18nObjects = Array.isArray(i18n)
+    ? i18n.map((t) => t.global)
+    : [i18n.global];
+
+  for (const obj of i18nObjects) {
+    if (obj?.dateLocale) {
+      return obj.dateLocale as Locale;
+    }
+  }
+  throw new Error("dateLocale not found.");
+}
```

Comment:
Is it fine to have a separate hook for this, or should it be part of `useI18n()`?

Reply:
Would be nice to be part of the same API, but unfortunate if it makes the `translate` more complex. So a separate hook works for me here.